tools/blobtoolkit/blobtoolkit.xml

```diff
@@ -0,0 +1,116 @@
+<tool id="blobtoolkit" name="BlobToolKit" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="21.05">
+    <description>genome assembly QC</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro='requirements' />
+    <command detect_errors="exit_code"><![CDATA[
+        #if $mode_conditional.selector == 'create'
+            #if $mode_conditional.id_conditional.selector == 'enabled'
+                mkdir -p './taxdump' &&
+                ln -s '${mode_conditional.id_conditional.taxdump}' 'taxdump.tar' &&
+                tar -xf 'taxdump.tar' -C './taxdump' &&
+            #end if
+            ln -s '${mode_conditional.fasta}' 'assembly.fasta' &&
+            #if $mode_conditional.meta
+                ln -s '${mode_conditional.meta}' 'assembly.yaml' &&
+            #end if
+            blobtools create 
+            --fasta 'assembly.fasta'
+            #if $mode_conditional.meta
+                --meta 'assembly.yaml'
+            #end if
+            #if $mode_conditional.id_conditional.selector == 'enabled'
+                --taxid $mode_conditional.id_conditional.taxid
+                --taxdump './taxdump'
+            #end if
+            ./Blobdir
+            && tar -zcf ./Blobdir.tgz ./Blobdir
+        #else if $mode_conditional.selector == 'add'
+            mkdir -b './Blobdir' &&
+            tar -zxf '${mode_conditional.blobdir}' -C './Blobdir' &&
+            #if $mode_conditional.bed
+                mkdir -p './bed_files' &&
+                #import re
+                #for $i, $input in enumerate($mode_conditional.bed):
+                    #set $safename = re.sub('[^\w\-_]', '_', $input.element_identifier) + "." + str($i)
```

Comment:
This line is required because it is necessary to remove whitespaces (in case the user provides a filename that include any).